ios/FluentUI/Core/SwiftUI+ViewModifiers.swift

```diff
@@ -40,6 +40,30 @@ extension View {
             .onPreferenceChange(SizePreferenceKey.self,
                                 perform: action)
     }
+
+    /// Adds a large content viewer for the view
+    /// - Parameters
+    ///  - text: Optional String to display in the large content viewer.
+    ///  - image: Optional UIImage to display in the large content viewer.
+    /// - Returns: The modified view.
+    func showsLargeContentViewer(text: String?, image: UIImage?) -> AnyView {
```

Comment:
You can avoid the AnyView return here and use some View instead.

Reply:
It doesn't build with some View, since it can return 3 different types of views.

Reply:
If you use a `ViewModifier` as mentioned by @rdeassis then I _think_ you can get away with `some View`.